README.md

```diff
@@ -29,6 +29,14 @@ ember install ember-frost-sort
 | `sortParams` | `array` | `[{"direction: "asc/desc", "value": <attr-name>}]` |  Array that specifies the sort order. |
 | `onChange` | `string` | `<action-name>` | The action to call when the value of the select item changes. |
 
+## Testing with ember-hook
+The sort component is accessible using ember-hook with the top level hook name or you can access the internal components as well -
+* Add filter button hook - `$hook('<hook-name>-add-filter')`
```

Comment:
Sorry to come into this late, but `filter` is a less than ideal word choice here, since this is going to be frequently co-located with actual content filters - why not `sort`?